src/components/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-nested-ternary */
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getAll } from '../../utils/fetchClient';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    getAll()
+      .then((peopleFromServer) => {
+        setPeople(peopleFromServer);
+      })
+      .catch(() => {
+        setError(true);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, []);
+
+  const visiblePeople = people.map(person => {
+    const newPerson = { ...person };
+
+    newPerson.mother = people.find(mother => mother.name === person.motherName);
+    newPerson.father = people.find(father => father.name === person.fatherName);
+
+    return newPerson;
```

Comment:
no reassign is needed (bad practice)
```suggestion
    return {
      ...person,
      mother: people.find(mother => mother.name === person.motherName),
      father: people.find(father => father.name === person.fatherName);
    };
```